docs/examples/basic.rst

```diff
@@ -53,6 +53,15 @@ Functions
 
 .. _example_schema_create:
 
+Aggregate
+=========
+.. literalinclude::  ../../examples/aggregations.py
```

Comment:
Is it right to call the example "aggregate"? It mostly is about adding explicit group_by statements?

Reply:
What about ``Group By``?

Reply:
I think that's fine :smile: